api/mapping/test_serializers.py

```diff
@@ -0,0 +1,196 @@
+import os
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIRequestFactory
+from rest_framework.authtoken.models import Token
+from rest_framework.serializers import ValidationError
+from .models import Project, Dataset, ScanReport, VisibilityChoices
+from .serializers import ScanReportEditSerializer
+
+
+class TestScanReportEditSerializer(TestCase):
+    def setUp(self):
+        User = get_user_model()
+        self.admin_user = User.objects.create(
+            username="gandalf", password="onfwojeijfe"
+        )
+        self.author_user = User.objects.create(
+            username="frodo", password="owdjqwojdowjjwf"
+        )
+        self.non_admin_user = User.objects.create(
```

Comment:
Could you rename this please to `non_viewer_user` or similar, to denote that this is "even less access than a viewer".